### PR TITLE
Check if tests pass with new `watchgod`

### PR DESCRIPTION
**Pull request by Kludex**
There was a pre-release, and I want to check if everything is fine with `uvicorn` using it: https://github.com/samuelcolvin/watchgod/releases/tag/v0.10a1

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def get_packages(package):
     "httptools>=0.4.0",
     "uvloop>=0.14.0,!=0.15.0,!=0.15.1; " + env_marker_cpython,
     "colorama>=0.4;" + env_marker_win,
-    "watchgod>=0.6",
+    "watchgod==0.10a1",
```
Comment (br3ndonland) [inline setup.py:59]: The package has been renamed to `watchfiles` as of [v0.10](https://github.com/samuelcolvin/watchfiles/releases/tag/v0.10).

Comment (Kludex) [reply setup.py:59]: I have a horrible memory, I completely forgot about it. Thanks Brendon! :)

**Comment by Kludex** [comment]
As @br3ndonland pointed out, we need to replace `watchgod` by `watchfiles`. I'll do it tomorrow.

**Comment by Kludex** [comment]
More changes are needed. I'll be working on it anyway.

**Pull request closed without merging**